### PR TITLE
Insert chunk, get response to questions.

### PR DESCRIPTION
**Pull request by carolina-museum**
a script file to insert chunks and a script file to answer questions based on the inserted chunks. 

The file to find the best chunk size is my next task. I will combine those two files and make the outputs easier to read.

### CODE REVIEW

File: scripts/ask_query.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+)
+
+try:
+    learn_collection = chromadb_client.get_collection(name="learn")
+except ValueError:
+    print("There is no collection. I will make one for you!")
+    learn_collection = chromadb_client.create_collection(name="learn")
+
+
+router = APIRouter()
```
Comment (huyfififi) [inline scripts/ask_query.py:26]: Let's remove irrelevant operations. In this script, we don't create any API endpoints with FastAPI

File: scripts/ask_query.py [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+    headers=None,
+    settings=chromadb.config.Settings(),
+    tenant=chromadb.config.DEFAULT_TENANT,
+    database=chromadb.config.DEFAULT_DATABASE,
+)
+
+try:
+    learn_collection = chromadb_client.get_collection(name="learn")
+except ValueError:
+    print("There is no collection. I will make one for you!")
```
Comment (huyfififi) [inline scripts/ask_query.py:22]: 👍

**Comment by carolina-museum** [comment]
Is it updated now...?

**Review by huyfififi — APPROVED** [review]
👍

**Pull request merged**